prow/cmd/gerrit/main.go

```diff
@@ -94,6 +98,7 @@ func gatherOptions(fs *flag.FlagSet, args ...string) options {
 	fs.BoolVar(&o.dryRun, "dry-run", false, "Run in dry-run mode, performing no modifying actions.")
 	fs.StringVar(&o.tokenPathOverride, "token-path", "", "Force the use of the token in this path, use with gcloud auth print-access-token")
 	fs.IntVar(&o.inRepoConfigCacheSize, "in-repo-config-cache-size", 100, "Cache size for ProwYAMLs read from in-repo configs. Each host receives its own cache.")
+	fs.IntVar(&o.changeWorkerPoolSize, "change-worker-pool-size", 10, "Number of workers processing changes for each instance.")
```

Comment:
Let's default to 1 to avoid breaking someone else

Reply:
Seems like a good idea. Once we see that it works fine we can set the default back to 10.